config/assets/default.js

```diff
@@ -35,6 +35,9 @@ module.exports = {
       'modules/*/client/*.js',
       'modules/*/client/**/*.js'
     ],
+    img: [
+      'modules/**/**/img/**/*'
```

Comment:
You can still specify specific images, I'd rather not just let this bluntly run on any file in there because there may be user error.